File: Data/Entities/Call.cs
namespace API.Data.Entities
{
    public class Call
    {
        public int Id { get; set; }
        public int RegistrId { get; set; }
        public Registr?  Registr { get; set; }
        public int CustomerId { get; set; }
        public  Customer? Customer { get; set; }
        public DateTime DateTime { get; set; }
        public string  Theme { get; set; }
        public string Feedback { get; set; }
    }
}

File: src/app/modes/call.ts
import { Customer } from "./customer";
import { Registr } from "./registr";

export interface Call{
    id: number;
    registrId: number;
    registr: Registr|undefined;
    customerId: number;
    customer: Customer|undefined;
    dateTime: Date;
    theme: string;
    feedback: string;
}